wherescape/connectors/hubspot/python_hubspot_check_missing_columns.py

```diff
@@ -39,7 +40,7 @@ def create_table_rows(
 
 
 def get_ds_rows(wherescape_instance):
-    # Get the datastore rows for comparison with the new load rows
+    """Get the datastore rows for comparison with the new load rows"""
```

Comment:
[PEP 257](https://peps.python.org/pep-0257/) recommends that the first line of a docstring is written in the form of a command, ending in a period.

Reply:
maybe add: `This function assumes that existence of a  datastore table in a datastore schema with the name as a load table but with a different prefix`